### ISSUE TITLE
Orechid and Excavator can generate HO's packaged ores

### ISSUE THREAD

**Issue by Barhandar** [Haggle1996/RevolutionPack#551]
Since Packager is disabled and these blocks are supposed to be unobtainable, this is a somewhat easy fix.

```
<ore:oreIron>.remove(<HarderOres:dummyOreIron>);
<ore:oreGold>.remove(<HarderOres:dummyOreGold>);
<ore:oreTin>.remove(<HarderOres:dummyOreTin>);
<ore:oreCopper>.remove(<HarderOres:dummyOreCopper>);
<ore:oreLead>.remove(<HarderOres:dummyOreLead>);
#See below: not sure if it will work and by default HO-U is the only thing in oreUranium
//<ore:oreUranium>.remove(<HarderOres:dummyOreUranium>); 
<ore:oreSilver>.remove(<HarderOres:dummyOreSilver>);
<ore:oreNickel>.remove(<HarderOres:dummyOreNickel>);
<ore:oreAluminum>.remove(<HarderOres:dummyOreAluminum>);
<ore:oreOsmium>.remove(<HarderOres:dummyOreOsmium>);

#ReC doesn't normally register Pitchblende as oreUranium, leaving HO Uranium Ore as the only existing.
#The above removes it, leaving oreUranium empty. Possibly this will allow IE Excavator to get actual Uranium instead of fake one if uncommented, but I don't know if purging Excavator ore cache is possible (book still displays HO-U even after the script running).
//<ore:oreUranium>.add(<ReactorCraft:reactorcraft_block_ore:5>); #Endstone Pitchblende
//<ore:oreUranium>.add(<ReactorCraft:reactorcraft_block_ore:1>); #Stone Pitchblende
```

**Comment by Haggle1996** [comment]
Unfortunately, CraftTweaker/MineTweaker don't actually remove items from the ore dictionary. This is a known problem. Further, who cares? The packaged ore behaves identically to regular ore, as it was intended.

**Comment by Barhandar** [comment]
> Unfortunately, CraftTweaker/MineTweaker don't actually remove items from the ore dictionary. 

Using that script in singleplayer did remove OD entry on the packaged ores, and they stopped appearing in orechid output.

> Further, who cares?

It's ugly. I don't want to need to hide my automatic oregen because of it.

**Comment by Haggle1996** [comment]
Strange. The arrays inside Forge are add-only. You shouldn't be able to do this to the ore dictionary.

**Comment by Haggle1996** [comment]
Yeah, sorry. Tested your script; the items are still in the dictionary.

However, looking at the original issue on the MineTweaker issue tracker, there may be a workable alternative.

**Comment by Barhandar** [comment]
> Yeah, sorry. Tested your script; the items are still in the dictionary.

No, no they're not.  
It's not an "items are not removed from OD" error. It's "clients aren't synchronized with server correctly the first time" error, relogging to the server removes the visible fake entries. 

If you have a code to check for this, is it running clientside?

**Comment by Haggle1996** [comment]
I'm very sorry, but your script doesn't work as expected. However, as I mentioned before, both the Excavator and Orechid have specific minetweaker functions to remove these blocks. Furthermore, if someone manages to generate one of the forbidden ores through legitimate means I'm unaware of, I'd rather still have it in the dictionary so that they're processable, then dead broken blocks. I will fix this by modifying the orechid and excavator scripts.

**Comment by Barhandar** [comment]
> I will fix this by modifying the orechid and excavator scripts.

_Good luck_.
